### ISSUE TITLE
3.5 performance regression: 100% CPU usage

### ISSUE THREAD

**Issue by rofl0r** [adventuregamestudio/ags#771]
when loading Gemini Rue DEMO.exe with 3.4.3.1, the title screen consumes about 33% CPU, whereas 3.5.x (from ags3 branch) consumes 100%.

**Comment by ivan-mogilko** [comment]
This may be duplicate for #715

**Comment by ivan-mogilko** [comment]
Right, so, this started after f2ab428a596c67d950f2aed5fbd335f6ccc72e3f.
Note that on Windows you must also include c6d1601615b6dd8936a6db50861baa4bda279fc6 otherwise engine won't compile

**Comment by rofl0r** [comment]
good find. i suppose reversing that commit would be an appropriate fix.

**Comment by sonneveld** [comment]
This should be fixed in `ags3` now if you'd like to test it.  Still waiting on PR to update the 3.5.0 release branch.

**Comment by sonneveld** [comment]
Closing because duplicate of https://github.com/adventuregamestudio/ags/issues/715
